docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,285 @@
+# Registering and enrolling identities with a CA
+
+*Audience: organization administrators, node administrators*
+
+If you've read our topics on [identity](https://hyperledger-fabric.readthedocs.io/en/release-2.1/identity/identity.html) and [Membership Service Provider (MSP)](https://hyperledger-fabric.readthedocs.io/en/release-2.1/membership/membership.html) you're aware that in Hyperledger Fabric, Certificate Authorities are used to generate the identities assigned to admins, nodes, and users (client applications). While any Certificate Authority that can generate x.509 certificates can be used to create the public/private key pair that constitutes an identity, the Fabric CA can additionally generate the local and organizational MSP folder structures that are required by Hyperledger Fabric.
```

Comment:
Rather than pointing to specific branches I think we should use the {BRANCH} conref instead.

so change:
https://hyperledger-fabric.readthedocs.io/en/release-2.1/identity/identity.html
to

and 
https://hyperledger-fabric.readthedocs.io/en/release-2.1/membership/membership.html
to
https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/membership/membership.html
https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/identity/identity.html